web/src/hooks/useCityContentParams.ts

```diff
@@ -2,7 +2,7 @@ import { useParams } from 'react-router-dom'
 
 const useCityContentParams = (): { cityCode: string; languageCode: string } => {
   const params = useParams()
-  return { cityCode: params.cityCode!, languageCode: params.languageCode! }
+  return { cityCode: params.cityCode as string, languageCode: params.languageCode as string }
```

Comment:
Example:
```suggestion
  // This hook should only be used in city content routes where it is guaranteed that both city and language code are set
  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
  return { cityCode: params.cityCode as string, languageCode: params.languageCode as string }
```

Reply:
I also prefer this solution. If you use the typecast it becomes intransparent, that there is a possibility that the type is not defined. @steffenkleinle I think you messed up the suggestion ^^ (as you still have the typecast)

```suggestion
  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
  return { cityCode: params.cityCode as string, languageCode: params.languageCode! }
```

Reply:
True 🙈 You also still have one typecast though haha @sarahsporck

Reply:
😂 I am working my way through the typecasts and will fix them, no need to fix the example further :D